sites/public/.env.template

```diff
@@ -37,6 +37,8 @@ SHOW_PWDLESS=TRUE
 # Note: BLOOM_API_BASE must match EXTERNAL_API_BASE in tasks/import-listings
 BLOOM_API_BASE=https://hba-dev-proxy.herokuapp.com
 GOOGLE_MAPS_API_KEY=SUPER_SECRET_KEY
+GOOGLE_MAPS_MAP_ID=
```

Comment:
This is a new variable required by updating to the latest map versions. There is a new way that you customize map settings (like whether or not it should show road names, or certain place icons) where you import those settings from a map id.